### PR TITLE
Shubham tyagi/feature update spanish home and pages

### PR DESCRIPTION
**Pull request by shubham-tyagi**
**Description**

This is a duplicate PR of PR #4320 So that I could solve the merge conflicts.

**Notes for Reviewers**

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 6e3fc9600460eca6a1383813315444b2f09c8692 at: https://61994795631a90952f9db74f--meshery-docs-preview.netlify.app

**Comment by shubham-tyagi** [comment]
Hey @leecalcote, I guess you forgot to sign off your commit. What could be done now 🤔 ?

**Comment by warunicorn19** [comment]
Magic ig @shubham-tyagi

**Comment by shubham-tyagi** [comment]
haha 😂 @warunicorn19. But I actually don't know whether should the merging should be reverted and lee would add the signoff or the branch could be merged without signoff?

**Comment by Mr-DG-Wick** [comment]
@shubham-tyagi I think as long as we have no issues with the build, the DCO problem should not be an issue :)

**Comment by shubham-tyagi** [comment]
Yeah, that is also right I guess @Mr-DG-Wick, but @warunicorn19 solved the DCO issue as well. Magician doing the magic haha.

File: docs/pages/es/guides/sample-apps.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+
+<a href="{{ site.baseurl }}/assets/img/adapters/sample-apps/httpbin.png"><img alt="httpbin" style="width:500px;height:auto;" src="{{ site.baseurl }}/assets/img/adapters/sample-apps/httpbin.png" /></a>
+
+### [Linkerd Books](https://github.com/BuoyantIO/booksapp)
+
+Linkerd Books es una aplicación de muestra basada en Ruby. Está diseñado para demostrar las diversas propuestas de valor, incluida la depuración, la observabilidad y el monitoreo de su malla de servicios. Puede usarse para determinar la eficiencia de su malla y para depurar.
+
+<a href="{{ site.baseurl }}/assets/img/adapters/sample-apps/linkerd-books.png"><img alt="Linkerd Books" style="width:500px;height:auto;" src="{{ site.baseurl }}/assets/img/adapters/sample-apps/linkerd-books.png" /></a>
+
+### [Hipster](https://github.com/GoogleCloudPlatform/microservices-demo)
```
Comment (warunicorn19) [inline docs/pages/es/guides/sample-apps.md:73]: Hipster is now called Online Boutique

File: docs/pages/es/guides/sample-apps.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+
+Hipster es una aplicación nativa de la nube de muestra, originalmente creada por Google. Se compone de 10 microservicios y se puede utilizar para mostrar y trabajar con Kubernetes, Istio, gRPC y OpenCensus. En la implementación, ejecuta una aplicación de demostración de comercio electrónico basada en la web, un ejemplo de la cual se puede ver a continuación:
+
+<a href="{{ site.baseurl }}/assets/img/adapters/sample-apps/hipster-shop.png"><img alt="Hipster" style="width:500px;height:auto;" src="{{ site.baseurl }}/assets/img/adapters/sample-apps/hipster-shop.png" /></a>
```
Comment (warunicorn19) [inline docs/pages/es/guides/sample-apps.md:77]:
```suggestion
<a href="{{ site.baseurl }}/assets/img/adapters/sample-apps/online-boutique.png"><img alt="Online Boutique" style="width:500px;height:auto;" src="{{ site.baseurl }}/assets/img/adapters/sample-apps/online-boutique.png" /></a>

```

**Review by warunicorn19 — CHANGES_REQUESTED** [review]
Thanks for taking this up @shubham-tyagi, suggested a few changes.

**Comment by shubham-tyagi** [comment]
@warunicorn19 I thought that all the files are pre-approved and only merge conflicts are need to be solved 😂. But I will be more than happy to apply all the suggestions.

**Comment by warunicorn19** [comment]
@shubham-tyagi good luck with the million merge conflicts😂.

**Comment by warunicorn19** [comment]
oh wow, you already solved them.

**Comment by l5io** [comment]
🚀 Preview for commit b8d7a39c80dc13149bfe056c8d8822157055310f at: https://619a9413281d3aabeb51e6fe--meshery-docs-preview.netlify.app

**Comment by shubham-tyagi** [comment]
@warunicorn19 not a piece of cake, for sure 😂

**Comment by leecalcote** [comment]
A quick review from @alonsolopezr @jlcastrogro @jesuslerma will help.

**Comment by alonsolopezr** [comment]
Good work 👍🏼... in the first file, there are some observations to check and modify:

Link to https://619a9413281d3aabeb51e6fe--meshery-docs-preview.netlify.app/es/extensibility/adapters#:~:text=adaptadores%20de%20malla%20de%20servicios
In the first paragraph :
“¿Qué son los Adaptadores Meshery?
Los adaptadores permiten a Meshery interactuar con las diferentes mallas de servicio. Revise la lista de todos los disponibles adaptadores de malla de servicios. Ver los diagramas de Arquitectura Meshery para visualizar sobre como los adaptadores se relacionan a otros componentes Meshery.”

The title and text differs in the section:
“Descripción general y uso
La biblioteca consta de interfaces e implementaciones predeterminadas para la funcionalidad principal y común de un adaptador. También proporciona un mini-framework que ejecuta el servicio del adaptador gRPC, llamando a las funciones de los controladores inyectados por el código del adaptador. Esto se representa en un estilo similar a UML en la siguiente figura. La biblioteca se utiliza en todos los adaptadores de Meshery.

Adaptador Meshery para Malla de Servicio XXX

Proceso de Desarrollo
Con el CONTRIBUTING.md en mente, Comprenda que el desarrollo sigue el flujo de trabajo de solicitud de bifurcación y extracción habitual que se describe aquí, consulte también Proceso de GitHub. Al bifurcar, GitHub desactiva todos los flujos de trabajo. Es una buena práctica y seguro activarlos de manera que el código se valide en cada inserción. Esto requiere que el filtro de ramas para “al enviar” esté configurado en “**” para que se active también en las ramas que contengan “/” en su nombre. Las acciones se parametrizan utilizando secretos (consulte Estrategia de creación y lanzamiento). La imagen de Docker solo se crea y se envía a Docker Hub si se inserta una etiqueta y se configura la información de autenticación correspondiente. El único secreto que debe establecerse en cada bifurcación es GO_VERSION, especificado en la estrategia de compilación y lanzamiento; de lo contrario, se usa la versión predeterminada de la acción correspondiente.
”


In English it has the text:
“Overview and Usage
The library consists of interfaces and default implementations for the main and common functionality of an adapter. It also provides a mini-framework that runs the gRPC adapter service, calling the functions of handlers injected by the adapter code. This is represented in an UML-ish style in the figure below. The library is used in all of Meshery’s adapters.

Contributing to Meshery Adapters
With the CONTRIBUTING.md in mind, understand that development follows the usual fork-and-pull request workflow described here, see also GitHub Process. On forking GitHub deactivates all workflows. It is safe and good practice to activate them such that the code is validated on each push. This requires that branches filter for “on push” is set to ‘**’ to be triggered also on branches containing ‘/’ in their name. The actions are parameterized using secrets (see Build & Release Strategy). The Docker image is only built and pushed to Docker Hub if a tag is pushed and the corresponding authentication information is configured. The only secret that should be set in each fork is GO_VERSION, specified in Build & Release Strategy, otherwise, the corresponding action’s default version is used.”

And after that too…
Maybe take on the actual English page content, to produce the Spanish version

**Comment by shubham-tyagi** [comment]
@alonsolopezr can you have a look again?

**Comment by shubham-tyagi** [comment]
@warunicorn19 can you do that DCO magic again?

**Comment by l5io** [comment]
🚀 Preview for commit 19a1c16be69c7ee98e797930f9288717f6d2ff69 at: https://61a113b3e5f3f24017442262--meshery-docs-preview.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit e7338490c046e1f0e4dcc57b0451b3b080ab0f45 at: https://61a26dfbe5f3f25e8a4423e4--meshery-docs-preview.netlify.app

File: docs/pages/es/reference/index.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+list: exclude
+---
+
+Referencias para utilizar las diversas funciones y componentes de Meshery.
+
+{% assign sorted_reference = site.pages | sort: "type" | reverse %}
+
+<ul>
+    {% for item in sorted_reference %}
+    {% if item.type=="Reference" and item.lang == "es" -%}
```
Comment (warunicorn19) [inline docs/pages/es/reference/index.md:15]:
```suggestion
    {% if item.type=="Reference" and item.lang == "es" and item.list!="exclude" -%}

```

**Review by warunicorn19 — CHANGES_REQUESTED** [review]
@shubham-tyagi really good stuff, one thing I noticed is the GKE file has been changed to EKS, any particular reason?? Due to this there is also a repetition of `EKS` in the menu under `Plataformas Soportadas`

**Comment by l5io** [comment]
🚀 Preview for commit 38890d697b911bcd5cf50bb917404b1386762114 at: https://61ab038ed8ecfa7aee3f4dcb--meshery-docs-preview.netlify.app

**Comment by shubham-tyagi** [comment]
@warunicorn19 I actually made that mistake while solving the merging conflicts, Thanks for pointing it out.

**Comment by l5io** [comment]
🚀 Preview for commit 1d01e9433e3b15f379656d5bce09d4ec794e1da5 at: https://61ab04ce607f188b68e9fdaf--meshery-docs-preview.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 1f55643a3204b111c8f58d091be92d6e986da3d0 at: https://61ab068bd8ecfa7d453e6779--meshery-docs-preview.netlify.app

**Review by warunicorn19 — DISMISSED** [review]
hey @MarioArriaga92 could you please take a quick look at this, I think everything looks good, but I can't read Spanish😅.

**Comment by l5io** [comment]
🚀 Preview for commit c2bb7f557ae2b995c2108f36fde894392fc11e69 at: https://61adf94eabedb22a3ff9eb1a--meshery-docs-preview.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 169025d519b5b3e06298fe73c1ccad8bcc400dec at: https://61adfbeb6e9eaa0de23ea67a--meshery-docs-preview.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 9f7dad2c43cf905e542d9f4837e4940a1638b5f7 at: https://61adfcc7d8ecfaad7e3e67fb--meshery-docs-preview.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 70ecefde4036b4c1be0b265257d9b0cc4708cf1e at: https://61adfdadcbaf510c4c18d496--meshery-docs-preview.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit cf2e4e019debbda4a8a62a93d50e80ce875a5ef0 at: https://61adff8d6e9eaa12f53ea671--meshery-docs-preview.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit cf2e4e019debbda4a8a62a93d50e80ce875a5ef0 at: https://61adff9c607f189f33e9fd9f--meshery-docs-preview.netlify.app

**Review by warunicorn19 — APPROVED** [review]
LGTM!

**Pull request merged**